browser/extensions/api/brave_rewards_api.cc

```diff
@@ -1166,33 +1166,33 @@ BraveRewardsGetExternalWalletFunction::Run() {
   std::unique_ptr<brave_rewards::GetExternalWallet::Params> params(
     brave_rewards::GetExternalWallet::Params::Create(*args_));
 
-  rewards_service->GetExternalWallet(
-      params->type,
+  rewards_service->GetUpholdWallet(
       base::BindOnce(
-          &BraveRewardsGetExternalWalletFunction::OnExternalWalet,
+          &BraveRewardsGetExternalWalletFunction::OnGetUpholdWallet,
           this));
   return RespondLater();
 }
 
-void BraveRewardsGetExternalWalletFunction::OnExternalWalet(
+void BraveRewardsGetExternalWalletFunction::OnGetUpholdWallet(
```

Comment:
Are we keeping the class name here as `BraveRewardsGetExternalWalletFunction`? I was expecting to see `BraveRewardsGetUpholdWalletFunction`...

Reply:
yeah I kept UI facing calls the same and just do under the hood changes. I do not know what UI logic will look like when we implement new UI/UX. To prevent multiple refactors I left it so that we do it when we implement second wallet